website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -0,0 +1,372 @@
+    },
+    {
+      "ServiceName": "db",
+      "Datacenters": ["dc1"]
+    }
+  ],
+  "NodeIdentities": [
+    {
+      "NodeName": "node-1",
+      "Datacenter": "dc2"
+    }
+  ]
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+During the authorization process, the following policies for the `web` and `db` services will be generated and linked to the token:
+
+<CodeBlockConfig filename="web-policy.hcl">
+
+```hcl
+# Allow the service and its sidecar proxy to register into the catalog.
+service "web" {
+	policy = "write"
+}
+service "web-sidecar-proxy" {
+	policy = "write"
+}
+
+# Allow for any potential upstreams to be resolved.
+service_prefix "" {
+    policy = "read"
+}
+node_prefix "" {
+    policy = "read"
+}
+```
+
+</CodeBlockConfig>
+
+Per the `ServiceIdentitites.Datacenters` configuration, the `db` policy is scoped to resources in the `dc1` datacenter.
+
+<CodeBlockConfig filename="db-policy.hcl">
+
+```hcl
+# Allow the service and its sidecar proxy to register into the catalog.
+service "db" {
+	policy = "write"
+}
+service "db-sidecar-proxy" {
+	policy = "write"
+}
+
+# Allow for any potential upstreams to be resolved.
+service_prefix "" {
+    policy = "read"
+}
+node_prefix "" {
+    policy = "read"
+}
+```
+
+</CodeBlockConfig>
+
+## Node Identities
+
+<!-- -> Added in Consul 1.8.1 -- remove and lean on doc version? -->
+
+You can specify a node identity when configuring roles or linking tokens to policies. _Node_ commonly refers to a Consul agent, but a node can also be a physical server, cloud instance, virtual machine, or container. 
+
+Node identities enable you to quickly construct policies for nodes, rather than manually creating identical polices for each node. They are used during the authorization process to automatically generate a policy for the node(s) specifed. You can specify the token linked to the policy in the [`acl_tokens_agent`](/docs/agent/options#acl_tokens_agent) field when configuring the agent. 
+
+### Node Identity Specification
+
+Use the following syntax to define a node identity:
+
+<CodeTabs>
+
+```json
+{
+  "NodeIdentities": [
+    {
+      "NodeName": "<node name>",
+      "Datacenters": ["<datacenter name>"]
+    }
+  ]
+}
+```
+
+
+```hcl
+NodeIdentities = {
+  NodeName = "<node name>"
+  Datacenters = ["<datacenter name>"]
+}
+```
+
+</CodeTabs>
+
+- `NodeIdentities`: Declares a node identity block.
+- `NodeIdentities.ServiceName`: String value that specifies the name of the node you want to associate with the policy.
+- `NodeIdentitites.Datacenters`: Array that specifies the names of datacenters in which the node identity applies. This field is optional.
+
+Refer to the the [API documentation for roles](/api/acl/roles#sample-payload) for additional information and examples.
+
+-> **Consul Enterprise Namespacing** - Node Identities can only be applied to tokens and roles in the `default` namespace. The generated policy rules allow for `service:read` permissions on all services in all namespaces.
```

Comment:
I understood that as a limitation. I'll follow up with someone after pushing these changes

Reply:
This is probably because nodes are scoped to admin partitions, not to namespaces. For that reason, it wouldn't make sense to be able to specify a node identity in a namespace... except the special `default` namespace which is a catch-all for things not added to a user-created namespace (since every token must has a namespace specified).

How to communicate this in a way that's clear to the reader is a different question. I agree that a reader may ask the question @karl-cardenas-coding posed.